pkg/asset/manifests/infrastructure.go

```diff
@@ -80,6 +80,12 @@ func (i *Infrastructure) Generate(dependencies asset.Parents) error {
 		},
 	}
 
+	if *installConfig.Config.ControlPlane.Replicas == 1 {
```

Comment:
Both `ControlPlane` and `Replicas` are pointers. Is it possible for either to be `nil` at this point in the code?

In the enhancement, @hexfusion [points out](https://github.com/openshift/enhancements/pull/560/files#r553001390) that we may want this check to be `< 3` instead of `== 1`. I didn't find any validation that prevents a user from setting replicas to 2, for example. Maybe we're guarded in a way that I just didn't find?

Reply:
Did we decide what the desired behavior of the cluster in the case of 2 control nodes is?
Are we expecting all the operators to behave the same in the case of 1 node or 2 nodes?
according to https://github.com/openshift/enhancements/pull/555#discussion_r553113665 if we will change the enum to something like 'SingleNode' then it explicitly one and not two, 
We need to get to a conclusion in the enhancement before updating the code here.